devday/src/app/modules/login/components/login/login.component.html

```diff
@@ -1,34 +1,34 @@
-<h2>Login</h2>
+<h2>Anmelden</h2>
 <form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
   <div class="form-group">
-    <label for="username">Username</label>
+    <label for="username">Benutzername</label>
     <input
       type="text"
       formControlName="username"
       class="form-control"
       [ngClass]="{ 'is-invalid': submitted && f.username.errors }"
     />
     <div *ngIf="submitted && f.username.errors" class="invalid-feedback">
-      <div *ngIf="f.username.errors.required">Username is required</div>
+      <div *ngIf="f.username.errors.required">Benutzername ist erforderlich</div>
     </div>
   </div>
   <div class="form-group">
-    <label for="password">Password</label>
+    <label for="password">Passwort</label>
```

Comment:
We should generate components for inputs where label is only shown when a label is given by input. Input type, state (readonly ...) should be configurable too.